numpy/core/_internal.py

```diff
@@ -17,7 +16,7 @@
 except ImportError:
     ctypes = None
 
-IS_PYPY = platform.python_implementation() == 'PyPy'
+IS_PYPY = 'PyPy' in sys.version
```

Comment:
Please use `'__pypy__' in sys.modules` which is more robust

Reply:
maybe, but the code snippet above was taken from CPython
https://github.com/python/cpython/blob/master/Lib/platform.py#L1032
So L

Reply:
meh. If I  file an issue / make a PR there to change that to something more robust, how will we know to resync the code here?

Reply:
ok, I am convinced  :)